posthog/test/test_migration_0420.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any
+
+from posthog.test.base import NonAtomicTestMigrations
+
+
+class UpdateSurveyResponseMigrationTest(NonAtomicTestMigrations):
+    migrate_from = "0419_remove_organization_available_features"
+    migrate_to = "0420_set_all_survey_responses_to_be_strings"
```

Comment:
oh, so you're writing migration no 420, is it? 😏 😂

Reply:
ugh I WISH